sunkit_spec/sunxspex_fitting/fitter.py

```diff
@@ -35,38 +35,38 @@
 
 from astropy.table import Table
 
-from sunxspex.logging import get_logger
-from sunxspex.sunxspex_fitting.data_loader import LoadSpec
-from sunxspex.sunxspex_fitting.instruments import rebin_any_array
-from sunxspex.sunxspex_fitting.likelihoods import LogLikelihoods
-from sunxspex.sunxspex_fitting.parameter_handler import Parameters, isnumber
-from sunxspex.sunxspex_fitting.photon_models_for_fitting import (  # noqa
+from sunkit_spec.logging import get_logger
+from sunkit_spec.sunxspex_fitting.data_loader import LoadSpec
+from sunkit_spec.sunxspex_fitting.instruments import rebin_any_array
+from sunkit_spec.sunxspex_fitting.likelihoods import LogLikelihoods
+from sunkit_spec.sunxspex_fitting.parameter_handler import Parameters, isnumber
+from sunkit_spec.sunxspex_fitting.photon_models_for_fitting import (  # noqa
     defined_photon_models,
     f_vth,
     thick_fn,
     thick_warm,
 )
-from sunxspex.sunxspex_fitting.rainbow_text import rainbow_text_lines
+from sunkit_spec.sunxspex_fitting.rainbow_text import rainbow_text_lines
 
 warnings.filterwarnings("ignore", category=RuntimeWarning)
 warnings.filterwarnings("ignore", category=np.VisibleDeprecationWarning)
 
 logger = get_logger(__name__, 'DEBUG')
 
-__all__ = ["SunXspex", "load"]
+__all__ = ["sunkit_spec", "load"]
 
 # Easily access log-likelihood/fit-stat methods from the one place, if SunXpsex class inherits this then data is duplicated
 LL_CLASS = LogLikelihoods()
 
 
-class SunXspex:
+class sunkit_spec:
```

Comment:
this class name probably shouldnt be the same name as the package

Reply:
I agree with this as well but I'm not sure what a good replacement would be. I can only think of something like `Fitter` but I'm not sure that's much better. Also, just to double check, was the name that won the poll `sunkit-spec` or `sunkit-spex`?

Reply:
I've renamed it `Fitter` for the moment again not a final name just temporary work around.